lib/generators/install/templates/microservices_engine.rb

```diff
@@ -0,0 +1,22 @@
+require 'net/http'
+config_file = '../mse_router_info.yml'
+puts("[MSE] > No router configuration YAML file found") and return if
```

Comment:
Doesn't that raise an Exception though? I just want to tell the user that the config isn't found and quit out (although I don't think `return` works outside of a method now that I think about it, so raise may be the better option)